rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -251,6 +233,13 @@ components:
           minLength: 1
           maxLength: 255
           example: 30f14c6c1fc85cba12bfd093aa8f90e999
+        statusUri:
```

Comment:
Since no one is actually using the response right now, I was thinking that we should just return an empty response for now. We can always update when someone needs it and we know exactly what we want they need.